cluster.yml

```diff
@@ -123,3 +123,9 @@
   roles:
     - { role: kubespray-defaults}
     - { role: kubernetes-apps, tags: apps }
+
+- hosts: kube-master
+  any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
+  roles:
+    - { role: kubespray-defaults}
+    - { role: kubernetes-apps/helm, when: helm_enabled, tags: ["apps", "helm"]}
```

Comment:
line 125 will already include helm if helm_enabled. This play is completely unnecessary.